lib/rules/no-multiple-objects-in-class.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * @author tyankatsu <https://github.com/tyankatsu0105>
+ * See LICENSE file in root directory for full license.
+ */
+'use strict'
+
+// ------------------------------------------------------------------------------
+// Requirements
+// ------------------------------------------------------------------------------
+
+const { defineTemplateBodyVisitor } = require('../utils')
+
+// ------------------------------------------------------------------------------
+// Rule Definition
+// ------------------------------------------------------------------------------
+
+/**
+ * count ObjectExpression element
+ * @param {VAttribute} node
+ * @return {number}
+ */
+function countObjectExpression(node) {
+  return node.value.expression.elements.filter(
+    (element) => element.type === 'ObjectExpression'
+  ).length
+}
+
+/**
+ * Remove `parent` proeprties from the given AST.
+ * @param {string} key The key.
+ * @param {any} value The value of the key.
+ * @returns {any} The value of the key to output.
+ */
+function replacer(key, value) {
```

Comment:
Did you use this for debugging?
Now you can use [astexplorer.net](https://astexplorer.net/).

![image](https://user-images.githubusercontent.com/16508807/85488202-df442480-b608-11ea-99c1-c155ca71dce3.png)

Reply:
wow it's mistake sorry

Reply:
No problem.